lib/shopify-cli/shopifolk.rb

```diff
@@ -0,0 +1,82 @@
+require 'fileutils'
+
+module ShopifyCli
+  ##
+  # ShopifyCli::Shopifolk contains the logic to determine if the user appears to be a Shopify employee
```

Comment:
Let's say "staff" rather than employee, since some people will be contractors rather than employees?